Gopkg.toml

```diff
@@ -12,6 +12,7 @@ required = [
   "github.com/knative/test-infra/scripts",
   "github.com/knative/test-infra/tools/dep-collector",
   "github.com/pkg/errors",
+  "github.com/tsenart/vegeta",
```

Comment:
You shouldn't need any of the changes here to pull in vegeta.  I used it in Serving because the only reference was in yaml.

Reply:
Removed.